drs3/pubsub/schemas/non_staged_file_requested.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$id": "https://raw.githubusercontent.com/ghga-de/milestones-docs/main/milestone_1/api_definitions/async_topics/non_staged_file_requested.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "additionalProperties": false,
+  "description": "This event type is triggered when a user requested to download a file from the DRS3 service that is not yet staged in the Outbox storage.",
+  "properties": {
+    "file_id": {
+      "description": "The public ID of the file (as generated by the Metadata Repository service)",
+      "type": "string"
+    },
+    "grouping_label": {
+      "description": "This is a label that might be use to group multiple files together. E.g. the study id.",
+      "type": "string"
+    },
+    "md5_checksum": {
+      "description": "The md5-checksum of the file content.",
+      "type": "string"
+    },
```

Comment:
You are right by not using these fields. They probably should be removed from the schema entirely. I will take care of that.